### PR TITLE
Added proxying of custom attributes.

### PR DESCRIPTION
**Pull request by piotr-dobrogost**

### CODE REVIEW

File: kotti_content_proxy/tests/test_models.py [hunk 1]
```diff
@@ -1,18 +1,35 @@
...
+            proxied_attrs=['some_attribute', 'another_attribute'],
+            title=u'Proxy',
+            description=u'This is a proxy')
     db_session.flush()
 
-    assert root['proxy'].proxied_object == doc
+    assert proxy.some_attribute == foo.some_attribute
+    with pytest.raises(AttributeError) as excinfo:
+        proxy.another_attribute
+    assert excinfo.value.message == "'Document' object has no attribute " \
```
Comment (disko) [inline kotti_content_proxy/tests/test_models.py]: Long strings are the one exception I make to the 80 chars rule.  Just append "  # noqa" to the end of the line to make it pass PEP8 checks.

**Comment by disko** [comment]
Could please add a changenote?  Other than that :+1:

**Comment by piotr-dobrogost** [comment]
Done. I also bumped version and changed its format.

**Comment by disko** [comment]
Great, thanks!

**Pull request merged**